examples/mnist_export_v2/export_test.py

```diff
@@ -0,0 +1,42 @@
+import tensorflow as tf
+import tensorflow_hub as hub
+import unittest
+import tempfile
+import os
+import export
+
+TEST_DATA_DIR = "examples/mnist_export_v2/test_data"
+
+
+class TFHubMNISTTest(tf.test.TestCase):
+  def setUp(self):
+    file_ = tf.gather(tf.io.matching_files("%s/*.jpg" % TEST_DATA_DIR), 0)
+    self.test_image = tf.expand_dims(
+        tf.image.decode_jpeg(
+            tf.io.read_file(file_)), 0)
+    self.test_label = int(str(file_.numpy()).split("/")[-1].split(".jpg")[0])
+
+  def test_model_exporting(self):
+    export.train_and_export(epoch=1, batch_size=64, export_path="%s/model/1" % self.get_temp_dir())
+    self.assertTrue(os.listdir(self.get_temp_dir()))
+
+  def test_empty_input(self):
+    if not os.path.exists("%s/model/1" % self.get_temp_dir()):
```

Comment:
Please don't call another test from a test and don't rely on anything produced by a different test.

Same below.

Reply:
`test_empty_input()` is loading the graph exported during the test and checking if it is running on empty input. So, as a matter of fact it does depend on the test model exported during the training. Can you suggest me some ways to get around this dependency? One way might be to look if the graph exists and if it doesn't, I can call that train_and_export function again from the test. However `test_model_exporting()` exactly does this thing. so to make things modular I did this. Even i think the tests shouldn't be interdependent, and I'm open to suggestions to get around this.